### PR TITLE
Добавил команду выгрузки определенной версии из хранилища конфигурации.

### PR DESCRIPTION
**Pull request by pumbaEO**

### CODE REVIEW

File: src/Классы/МенеджерКонфигуратора.os [hunk 1]
```diff
@@ -348,6 +348,48 @@
...
+		ХранилищеКонфигурации = Новый МенеджерХранилищаКонфигурации();
+		ХранилищеКонфигурации.УстановитьУправлениеКонфигуратором(УправлениеКонфигуратором);
+		ХранилищеКонфигурации.УстановитьПараметрыАвторизации(ЛогинАдминистратора, ПарольАдминистратора);
+	
+		ХранилищеКонфигурации.УстановитьКаталогХранилища(ПутьХранилища);
+	
+		Попытка
+			ХранилищеКонфигурации.СохранитьВерсиюКонфигурацииВФайл(ВерсияХранилища, ПутьКФайлу);
+			
+			Лог.Информация("Выгрузка конфигурации из хранилища завершено");
```
Comment (artbear) [inline src/Классы/МенеджерКонфигуратора.os]: Правильно `завершена`!

File: src/Классы/КомандаСохранитьВерсиюХранилищаВФайл.os [hunk 2]
```diff
@@ -0,0 +1,86 @@
+///////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Подключение ИБ к хранилищу конфигурации 1С.
```
Comment (artbear) [inline src/Классы/КомандаСохранитьВерсиюХранилищаВФайл.os]: Предлагаю `Выгрузить файл конфигурации определенной версии из хранилища 1С`
как в описании метода ниже в PR

File: src/Классы/КомандаСохранитьВерсиюХранилищаВФайл.os [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Перем Лог;
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Прикладной интерфейс
+
+Процедура ЗарегистрироватьКоманду(Знач ИмяКоманды, Знач Парсер) Экспорт
+
+	ТекстОписания = 
+		"     Сохранить версию хранилища в файл cf.
```
Comment (artbear) [inline src/Классы/КомандаСохранитьВерсиюХранилищаВФайл.os]: Предлагаю `Выгрузить файл конфигурации определенной версии из хранилища 1С`
как в описании метода ниже в PR

**Comment by silverbulleters-ci-admin** [comment]
SonarQube analysis reported 46 issues
* ![MAJOR][MAJOR] 17 major
* ![MINOR][MINOR] 26 minor
* ![INFO][INFO] 3 info

#### Top 10 issues

1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L18](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L18): Rephrase this sentence to make it shorter. Actual size: 106 characters. Maximum expected size: 100 characters. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Astep-sentence-length)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L22](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L22): Reduce the number of steps (13, greater than 12 allowed). [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Amax-number-steps)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L23](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L23): Rephrase this sentence to make it shorter. Actual size: 107 characters. Maximum expected size: 100 characters. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Astep-sentence-length)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L28](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L28): Reduce the number of steps (13, greater than 12 allowed). [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Amax-number-steps)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L29](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L29): Rephrase this sentence to make it shorter. Actual size: 107 characters. Maximum expected size: 100 characters. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Astep-sentence-length)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L29](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L29): Merge all the When steps or split the scenario up in multiple scenarios. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Aone-single-when-per-scenario)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L30](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L30): Rephrase this sentence to make it shorter. Actual size: 178 characters. Maximum expected size: 100 characters. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Astep-sentence-length)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L34](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L34): Reduce the number of steps (14, greater than 12 allowed). [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Amax-number-steps)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L35](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L35): Rephrase this sentence to make it shorter. Actual size: 107 characters. Maximum expected size: 100 characters. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Astep-sentence-length)
1. ![MAJOR][MAJOR] [РаботаСХранилищем.feature#L35](https://github.com/silverbulleters/vanessa-runner/blob/0b69e3bdb587ad7c6d11c1c6cf85bbe4ecc90223/features/РаботаСХранилищем.feature#L35): Merge all the When steps or split the scenario up in multiple scenarios. [rule](https://sonar.silverbulleters.org/coding_rules#rule_key=gherkin%3Aone-single-when-per-scenario)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**